docs/index.md

```diff
@@ -127,3 +127,11 @@ async def main():
 if __name__ == "__main__":
     asyncio.run(main())
 ```
+
+
+## Run args
+
+You can pass arbitrary `key=value` arguments to `agent.run` and they will be forwarded
+to the corresponding method used by the underlying framework.
+
+For example, you can pass `max_turns=30` when using the [OpenAI Agents SDK](https://openai.github.io/openai-agents-python/).
```

Comment:
Since this is a common setting across all(or most) of the frameworks I feel like maybe we should add this as a specific param? By allowing the passing of arbitrary key=val arg, it basically allows someone to bypass the anyagent api, which then may lock them into a specific framework. For common args like this maybe we should make a specific paramter for it.

Reply:
> For common args like this maybe we should make a specific paramter for it.

I thought about the same, but then it turns out that not all frameworks support it 😅 .

We could still update the docs to show a tab-style block with how to achieve the same in the frameworks that support it?